mobilecoind/src/service.rs

```diff
@@ -2158,6 +2158,17 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
                 RpcStatus::with_message(RpcStatusCode::INTERNAL, "no peers reachable".to_owned())
             })?;
 
+        let mut mcd_last_block_info = api::LastBlockInfo::new();
+        mcd_last_block_info.set_index(last_block_info.block_index);
```

Comment:
:thought_balloon: The way this `last_block_info` was derived seems to deviate from the previous use of `LastBlockInfo` being what consensus said it was, thus further justifying a separate one for mobilecoind